plugins/taofurigana/plugin.js

```
@@ -1,733 +1,1056 @@
+                    rubyElement &&
+                    rtElement.getFirst().equals(startContainer) &&
+                    !beforeCaret.replace(zeroWidthSpaceRegex, '').length
+                ) {
+                    editor.fire('lockSnapshot');
+                    try {
+                        var caretRange = new CKEDITOR.dom.range(editor.document);
+                        caretRange.moveToPosition(rubyElement, CKEDITOR.POSITION_BEFORE_START);
+                        selection.selectRanges([caretRange]);
+                    } finally {
+                        editor.fire('unlockSnapshot');
+                        return true;
```

Biome (2.4.15)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)